service/src/main/resources/templates/ehr_observation_statement_effective_time_template.mustache

```diff
@@ -6,11 +6,14 @@
             {{{effectiveTime}}}
         </effectiveTime>
         <availabilityTime value="{{issued}}"/>
+        {{{value}}}
         <pertinentInformation typeCode="PERT">
             <sequenceNumber value="+1" />
             <pertinentAnnotation classCode="OBS" moodCode="EVN">
+                {{{valueQuantity}}}
```

Comment:
Is this `{{{valueQuantity}}}` used? It doesn't looks like it belongs. I think everything is prepended to the comment